.buildkite/common.py

```diff
@@ -180,16 +180,7 @@ def random_str(k: int):
 
 def ab_revision_build(revision):
     """Generate steps for building an A/B-test revision"""
-    # Copied from framework/ab_test. Double dollar signs needed for Buildkite (otherwise it will try to interpolate itself)
-    return [
-        f"commitish={revision}",
-        f"if ! git cat-file -t $$commitish; then commitish=origin/{revision}; fi",
-        "branch_name=tmp-$$commitish",
-        "git branch $$branch_name $$commitish",
-        f"git clone -b $$branch_name . build/{revision}",
-        f"cd build/{revision} && ./tools/devtool -y build --release && cd -",
-        "git branch -D $$branch_name",
-    ]
+    return [f"./tools/devtool -y build --rev {revision} --release"]
```

Comment:
Commenting about this commit in general. Would it not be simpler to just do this outside of devtool, i.e. as part of the pipeline?

I.e.

```sh
git checkout branch1
./tools/devtool build --release
cp -rvf <release> directory1
git checkout -
./tools/devtool build --release
cp -rvf <release> directory2
# Do sthg with directory1 and directory2
```

Reply:
I made it a devtool command because I wanted something that can be easily used outside of a pipeline. See the updated documentation in https://github.com/firecracker-microvm/firecracker/pull/4879/commits/a0925e8e30674e16892d8d26f66f8adec3d14292 for example